GitCommands/Git/Commands/GitCommandHelpers.cs

```diff
@@ -61,6 +61,17 @@ public static ArgumentString AddSubmoduleCmd(string remotePath, string localPath
             };
         }
 
+        public static ArgumentString AddWorktreeCmd(string path, string branch, bool force)
```

Comment:
Why do we need this for just displaying repo info?
It is used for setting up tests.

Reply:
I added this method similar to `AddSubmoduleCmd`. It's only used for tests.